### DISCUSSION TITLE
v2.7.1-RC

### DISCUSSION THREAD

**Discussion by Gkiokan** [Gkiokan/ps4-remote-pkg-sender#30] [Announcements]
Hi folks,

I am proudly presenting you the v2.7.1 with fully integrated HB-Store, Darkmode and Queue Scanner!  
Features that all of you wanted and now are working and making this app even better.  

HB-Store direct install Video on [instagram](https://www.instagram.com/p/CbxwMQJDPCX/). 
Queue Scanner in action on [instagram](https://www.instagram.com/p/Cb05rXDj785/). 

![PS4 Package Sender Main Application Windows](https://media.discordapp.net/attachments/933730584721780806/959908370565959750/new_cover_ported.jpg)

[![Full Changelog](https://img.shields.io/badge/Checkout%20-All%20Changelogs-yellow)](https://github.com/Gkiokan/ps4-remote-pkg-sender/blob/master/Changelog.md) 

[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/M4M082WK8)

<hr /><em>This discussion was created from the release <a href='https://github.com/Gkiokan/ps4-remote-pkg-sender/releases/tag/v2.7.1'>v2.7.1-RC</a>.</em>

**Comment by z3r001** [comment]
I have a raspberry pi without GUI an i like to use it from terminal over ssh

**Reply by Gkiokan** [reply]
This is an interesting use case. This is not supported yet, but could be in the future.  
One of the upcoming featuers are the "serve as host on www and install on friends ps4" will have a integrated dedicated server running with open ports, so I could take that and make it as an executable though commandline.  

But how would send the pkgs then to the ps4? You would still need an cmd interface or a menu for it that runs commands.  
Maybe creating a standalone server edition that could be installed though apt and brew is a better way.  

I will be back on the development route and think about it.  Feel free to open an Issue as [feature] if you want it, so ppl can vote on it, too.
